Assets/Scripts3D/Weapons/Weapon.cs

```diff
@@ -4,19 +4,49 @@
 
 public class Weapon : MonoBehaviour
 {
-
     [SerializeField] Camera FPCamera;
     [SerializeField] float Range = 100f;
     [SerializeField] float damage = 20f;
     [SerializeField] ParticleSystem MuzzleFlash;
+    [SerializeField] Transform EquippedWeapon;
```

Comment:
```suggestion
    [SerializeField]
    Transform EquippedWeapon;
```
Let's use this format for `SerializeField` as a convention

Reply:
Also just noticed this is a transform, maybe it should be called `EquippedWeaponPosition`?